components/shared/BreadcrumbsFilter.vue

```diff
@@ -4,11 +4,31 @@
       <NeoTag
         v-if="key === 'search'"
         :key="key"
-        class="control"
-        @close="removeSearch">
+        class="control a"
+        @close="removeBread('search')">
         {{ `${$t('general.search')}: ${value}` }}
       </NeoTag>
-      <NeoTag v-else :key="key" class="control" @close="closeTag(String(key))">
+      <NeoTag
+        v-else-if="key === 'min'"
+        :key="key"
+        class="control b"
+        @close="removeBread('min')">
+        {{ `${$t('Min')}:` }}
+        <CommonTokenMoney :value="value" />
+      </NeoTag>
+      <NeoTag
+        v-else-if="key === 'max'"
+        :key="key"
+        class="control c"
+        @close="removeBread('max')">
+        {{ `${$t('Max')}:` }}
+        <CommonTokenMoney :value="value" />
+      </NeoTag>
+      <NeoTag
+        v-else
+        :key="key"
+        class="control d"
+        @close="closeTag(String(key))">
```

Comment:
Why can't this fit into?
```vue
<NeoTag v-else :key="key" class="control" @close="closeTag(String(key))">
```

Reply:
maybe to add label?

Reply:
yeah label is different for them, I'll see if it can be merged in some way